TACC/Core-Styles
# feat: change accent color from purple to blue

## Overview

Change the `--global-color-accent--…` from purple to blue.

<details>Originally, the purple was form and from Portal. (CMS used Bootstrap colors.) CMS inherited Portal purple. But now the new TACC CMS uses blue. Designers agreed TACC blue will be the new default CMS accent color. I decided it is also the new base accent color. So, Portal and Docs will now overwrite it to use purple. This causes less CSS overwrite in CMS.</details>

## Related

- required by https://github.com/TACC/tup-ui/pull/203
- required by https://github.com/TACC/Core-CMS/pull/617
- required by https://github.com/TACC/TACC-Docs/pull/9

## Changes

- **moved** import of `components/c-recognition.css` to `core-styles.cms.css`
- **changed** demo link color (to match base accent color)
- **removed** load of `generics/color.css` from `core-styles.base.css`
- **added** color context switching for `_blue` (CMS, Demo) and `_purple` (unused)[^1]
- **changed** Portal and Docs to explicitly use purple accent colors in settings (to retain it)
- **changed** Base color settings to use blue accent color

[^1]: Portal and Docs do not use the patterns that change color context.

## Testing

0. `npm run build:css && npm run start`
1. Load [demo](http://localhost:3000).
2. Verify demo nav links are sapphire blue.
3. Open [Components > C-Button > Primary](http://localhost:3000/components/detail/c-button--primary).
4. Verify button backgrounds are sapphire blue.
5. Open [Elements > Links](http://localhost:3000/components/detail/links).
6. Verify button backgrounds are sapphire blue.
7. Verify successful test on clients:
    - https://github.com/TACC/tup-ui/pull/203
    - https://github.com/TACC/Core-CMS/pull/617
    - https://github.com/TACC/TACC-Docs/pull/9

## UI

| Buttons | Links |
| - | - |
| ![buttons](https://user-images.githubusercontent.com/62723358/229162429-cf45dc3d-84c4-415e-9b7d-87557536a960.png) | ![links](https://user-images.githubusercontent.com/62723358/229162416-0840469b-a30f-42e9-8768-48a7bd1f9575.png) |
